x-pack/plugins/fleet/server/services/agent_policies/package_policies_to_agent_permissions.ts

```diff
@@ -42,6 +52,12 @@ export async function storedPackagePoliciesToAgentPermissions(
 
       const pkg = packageInfoCache.get(pkgToPkgKey(packagePolicy.package))!;
 
+      // Special handling for Universal Profiling packages, as it does not use data streams _only_,
+      // but also indices that do not adhere to the convention.
+      if (pkg.name==='profiler_symbolizer') {
```

Comment:
nit: will be great to define a constant for that package here https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/common/constants/epm.ts